experiment/evaluator.py

```diff
@@ -195,13 +205,78 @@ def check_target(self, ai_binary, target_path: str) -> Optional[Result]:
       traceback.print_exc()
       return None
 
-  def _parse_libfuzzer_logs(self, log_handle) -> tuple[int, int, bool]:
-    """Parse libFuzzer logs."""
+  def _parse_stacks_from_libfuzzer_logs(self, lines: list[str]) -> list[str]:
+    """Parse stack traces from libFuzzer logs."""
+    # There can have over one thread stack in a log.
+    stacks = []
+
+    # A stack -> a sequence of stack frame lines.
```

Comment:
@DonggeLiu  we should add a TODO here to use our stack parsing from ClusterFuzz here instead, which is more tested and can do things like skipping inlined/common C lib functions.

Reply:
Yep! I presume this is the one?
https://github.com/google/clusterfuzz/blob/698d0bc6dba09b7af8d5acf3fe0941e909e43d57/src/clusterfuzz/_internal/crash_analysis/stack_parsing/stack_parser.py

@happy-qop Would you mind helping me add this in your new PR, given it is ready? Thanks!
```
# TODO (dongge): Use stack parsing from ClusterFuzz.
```

Reply:
Amazing, thanks sooooo much!

Reply:
How about addressing this under the same PR #198?
For example, having a new branch of the current [`stacktrace-fixing-exp`](https://github.com/google/oss-fuzz-gen/tree/stacktrace-fixing-exp) and merge it to #198 once it is ready.
Then we can run further experiments from there.